cumulus/parachains/integration-tests/emulated/chains/parachains/assets/asset-hub-westend/src/genesis.rs

```diff
@@ -90,6 +93,43 @@ pub fn genesis() -> Storage {
 					false,
 					ED,
 				),
+				// Ether
+				(
+					xcm::v5::Location::new(
+						2,
+						[xcm::v5::Junction::GlobalConsensus(EthereumNetwork::get())],
+					),
+					EthereumLocationsConverterFor::<[u8; 32]>::convert_location(
+						&xcm::v5::Location::new(
```

Comment:
nit: could move this to some global const `EthereumSovereignAcc` or some `parameter_types!` definition and reuse it rather than paste it verbosely all over the place

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/26